.github/workflows/rain.yml

```diff
@@ -19,9 +19,9 @@ jobs:
     name: Aggregate rain data from DWD radolan
     steps:
       - name: Harvester
-        uses: technologiestiftung/giessdenkiez-de-dwd-harvester@v2.4.1
+        uses: docker://technologiestiftung/giessdenkiez-de-dwd-harvester:v2.4.1
```

Comment:
Never saw this notion. Do you have any reference about this?

Reply:
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-using-a-docker-hub-action